stelaval/assignment4/IslandsCounterTest.java

```diff
@@ -0,0 +1,64 @@
+package assignment4;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public class IslandsCounterTest {
+  
+  @Test
+  public void countIslandsExample() {
+    IslandsCounter counter = new IslandsCounter();

```

Comment:
IslandsCounter is stateless, so it can safely be created either in `setUp()` method or directly as field in `IslandsCounterTest` class.